src/components/UserProfileHeader/UserProfileHeader.tsx

```diff
@@ -0,0 +1,96 @@
+import CloseIcon from '@mui/icons-material/Close';
+import { Box, Typography } from '@mui/material';
+import { ChangeEvent, useRef } from 'react';
+
+import { IUser } from '../../interfaces/IUser';
+import theme from '../../themes/theme';
+import {
+  StyledAvatar,
+  StyledBox,
+  StyledFileUploadIcon,
+  StyledIconButton,
+  StyledTypography
+} from './UserProfileHeader.styles';
+
+interface Props {
+  user?: IUser;
+  ableToEdit: boolean;
+  deleteAvatar: () => void;
+  uploadAvatar: (event: ChangeEvent<HTMLInputElement>) => void;
+}
+
+const getUserInitials = (user?: IUser) => {
+  if (!user) {
+    return '';
+  }
+
+  // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
It is better to disable esling rules in the eslintrc file if you dont need them.

Reply:
This rule is useful that's why I don't disable it in config file. But I can't avoid the error here